crates/sui-prover/tests/inputs/2.ok.move

```diff
@@ -0,0 +1,62 @@
+module 0x42::fb {
+  public fun foo() {
+    assert!(true);
+  }
+
+  public fun bar() {
+    foo();
+    assert!(true);
+  }
+}
+
+module 0x42::foo_specs {
+  use prover::prover::ensures;
+  use 0x42::fb::foo;
+
+  #[spec(prove, target = 0x42::fb::foo)]
+  public fun foo_spec() {
+    foo();
+    ensures(true); 
+  }
+}
+
+module 0x42::foo_specs_2 {
+  use prover::prover::ensures;
+  use 0x42::fb::foo;
+
+  #[spec(prove, target = 0x42::fb::foo)]
+  public fun foo_spec() {
+    foo();
+    ensures(true); 
+  }
+}
```

Comment:
let's have 2 different specs for `foo`, so it's possible to distinguish between them